ci/docker/runtime_functions.sh

```diff
@@ -1296,6 +1296,20 @@ nightly_scala_demo_test_cpu() {
     bash bin/run_im.sh
 }
 
+nightly_estimator_cnn_gpu() {
+    set -ex
+    cd /work/mxnet/tests/nightly/estimator
+    export PYTHONPATH=/work/mxnet/python/
+    python tests/nightly/estimator/test_estimator_cnn_gpu.py
```

Comment:
Is the `test_estimator_cnn_gpu.py` file path correct? The current shell location is already inside `/work/mxnet/tests/nightly/estimator` directory.

Reply:
fixed!